### DISCUSSION TITLE
Documentation for deploying with docker/docker-compose

### DISCUSSION THREAD

**Discussion by tkmamidi** [KarchinLab/open-cravat#304] [Usability & Feature Requests]
I see a Dockerfile but the documentation for deploying and working with various services (oc run and oc gui) is missing.
Any help is much appreciated!
Thanks!

**Comment by jasminebro** [comment]
Hi @tkmamidi! Is this (https://hub.docker.com/r/karchinlab/opencravat) the  webpage you looked at when you found the Dockerfile? There is a usage section that details installing modules, annotating variants, and Web GUI. Let me know if you need more help!

**Comment by tkmamidi** [comment]
This is great!
I'll try it out!
Thanks!
